tests/applepay_test.py

```diff
@@ -91,3 +92,37 @@ def test_signing_time_equals_current_time(token_fixture):
 
     # then the token should be considered valid.
     assert valid is True
+
+
+def test_valid_signing_time_data_is_logged(token_fixture, caplog):
+    # Given: a valid signature for a current time and threshold
+    signature = token_fixture['signature']
+    current_time = datetime(2014, 10, 27, 20, 51, 43, tzinfo=utc)
+    threshold = timedelta(hours=1)
+
+    # When we attempt to validate the signing time against the threshold,
+    with caplog.atLevel(logging.DEBUG):
```

Comment:
Correct, I'm more just curious about if there was a reason to use a context manager here (since setLevel seems as though it would accomplish the same thing)